wiki/Help_Centre/Client/en.md

```diff
@@ -0,0 +1,491 @@
+
+#### Are the score submission servers up?
+
+**The answer is almost always yes, but check the [osu! server status](https://status.ppy.sh) or the [@osustatus Twitter](https://twitter.com/osustatus) to make sure everything is running smoothly on our end.**
+
+If the servers are offline, do not close osu! until we've fixed the issues and the servers come back up, and the game will try to submit your score again for up to an hour before it gives up.
+
+### Can you upload a score I set for me?
+
+**Unfortunately, we cannot upload scores upon request.**
+
+If you were denied a score due to technical reasons or anything else, the best you can hope for is to repeat your display of skill and achieve it again. Sorry!
+
+### The notes seem out of sync or out of time with the music!
+
+**This can be due to sound processing software running on your machine, such as *Razer Surround Audio*. It could also be due to your `Universal Offset` being set incorrectly.**
+
+Try closing any extra programs which alter or influence the sound coming out of your computer. If this doesn't help, you could try to adjust your `Universal Offset`.
+
+Every individual hardware configuration has a particular `Universal Offset` that determines whether you hear the game's audio a little later or earlier.
+
+The default value works for most people, but if you find it isn't for you, you can change it by following these steps:
+
+1. Open osu!.
+2. Click the `Options` button on the main menu, or press `Ctrl` + `O`.
+3. Type `offset` into the quick-search bar. This will take you to the current universal offset your client is using.
+4. Use the slider to adjust your universal offset. Alternatively, try using the [offset wizard](/wiki/Options/Offset_Wizard) to help you with this process.
+
+If this still doesn't help, you may want to try updating your sound card or motherboard audio drivers.
+
+### My game looks glitchy, like the graphics are torn!
+
+**This is due to an effect called [screen tearing](https://en.wikipedia.org/wiki/Screen_tearing) and can be fixed by setting the in-game `Frame limiter` option to `VSync`.**
+
+This is a completely harmless (albeit ugly!) effect that happens because the game's refresh rate is not synced up with your monitor's refresh rate.
+
+You can also try playing in Windowed or Borderless mode, which will use your operating system's native VSync instead.
+
+Turning on VSync or playing in Windowed or Borderless mode will introduce a *slight* amount of input lag, unfortunately. For most people, this will not cause significant issues.
+
+Later NVidia graphics cards have a global option in the NVidia Control Panel to set the global refresh rate to a setting called 'Fast', which will also fix this issue.
+
+## Online features
+
```

Comment:
Assuming you haven't made any changes to the actual text under the headers, but while we're here, might as well add thematic break between these "section changes"